orc8r/cloud/docker/controller/Dockerfile

```diff
@@ -103,11 +103,13 @@ COPY src/magma/orc8r/cloud/docker/wait-for-it.sh /usr/local/bin
 RUN mkdir -p /var/opt/magma/envdir
 
 # Build artifacts
```

Comment:
General notes
- The goswagger-config.yml is part of swaggergen, not combine_swagger. So it needs to stay in its original spot. You could consider renaming the original location to get rid of the "apidocs" name from vcs filesystem, but in any case that file doesn't belong in the v1 pkg
- Where are we copying over the swagger ui static assets?

Reply:
1.) Ah I see, weird that the goswagger-config.yml was originally placed within apidocs/ but okay. I definitely agree that it doesn't belong under the v1 scope, I'll look to move it under swaggergen. 
2.) We copy over the swagger ui static assets within the Dockerfile::92. There wasn't any changes on it since we discussed to have them remain where they were right?